quickstart/overview.markdown

```diff
@@ -3,24 +3,24 @@ title: Overview of the ALICE O<sup>2</sup>
 layout: main
 ---
 
-This is an overview of the O<sup>2</sup> system. It is pretty rough but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
+This is an overview of the O<sup>2</sup> system. It is pretty rough, but it should give you an idea of the general architecture. Please refer to other documents such as the [TDR](https://cds.cern.ch/record/2011297).
 
 ## Requirements
 
-1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz what results in a 100 times more data than in 2010.
+1. After LS2 the ALICE experiment will have gone through a massive upgrade to handle minimum bias PbPb interactions at 50 kHz. This will results in a 100 times more data than in Run 2.
 
-2. Moreover, the physics topics addressed by the ALICE upgrade are characterised by a very small signal-to-noise ratio, requiring very large statistics, and a large background, which makes triggering techniques very inefficient, if not impossible.
+2. The physics topics addressed by the ALICE upgrade are characterised by a very small signal-to-noise ratio, requiring very large statistics, and a large background, which makes triggering techniques very inefficient, if not impossible.
 
-3. Finally, the TPC has an inherent rate <50Hz requiring a continuous, _non-triggered_, read-out.
+3. Finally, the TPC's inherent rate of ~50Hz requires a continuous, _non-triggered_, read-out.
 
 ## A new computing system : ALICE O<sup>2</sup>
 
- For these 3 main reasons, the amount of data to be handled is so large (3.4TB/s) that it is not possible to simply store it as we did during Run 1 and 2.
+ Due to above requirements the readout throughput increased to 3.4TB/s. Such amount of data is difficult to store as it was done during Run 1 and 2.
```

Comment:
"Given the above requirements, "
"increased" -> "increases" (we use present in the rest of the text).